pkg/controllers/management/auth/globalroles/fleetworkspace_binding_handler.go

```diff
@@ -0,0 +1,225 @@
+package globalroles
+
+import (
+	"fmt"
+	"reflect"
+
+	wrangler "github.com/rancher/wrangler/v2/pkg/name"
+
+	"github.com/hashicorp/go-multierror"
```

Comment:
multierror has the MPL2.0 license, which we prefer to avoid where possible. Could you use [errors.Join](https://pkg.go.dev/errors#Join)? I realize we use it in other places, but we should avoid it going forward
Something like this: https://github.com/rancher/webhook/blob/54bba7ed560004f40cd277aef921bdcd07802ea6/pkg/resources/management.cattle.io/v3/globalrolebinding/validator.go#L147

Reply:
done